dali/pipeline/operator/builtin/external_source.h

```diff
@@ -161,6 +162,22 @@ auto get_batch_size(const BatchContainer<Backend>& container) {
 
 }  // namespace detail
 
+/**
+ * @brief Options that can be configured when setting data for the External Source
+ *
+ * @param sync If SetExternalInputHelper should be blocking - waits until provided data is copied
+ *             to the internal buffer
+ * @param use_copy_kernel If true, a copy kernel will be used to make a
+ *                        contiguous buffer instead of cudaMemcpyAsync.
+ * @param no_copy_mode Select whether to use the parameter defined in the External Source or
+ *                     override the mode of operation forcing the copy or no-copy
+ */
+struct SetDataConfig {
```

Comment:
How about changing the name to `SourceDataConfig` or `ExternalSourceDataConfig`? I feel, that `Set` here doesn't really fit.

Reply:
Done with ExtSrcSettingMode